src/main/scala/nl.knaw.dans.easy.deposit/EasyDepositApiApp.scala

```diff
@@ -183,7 +214,7 @@ class EasyDepositApiApp(configuration: Configuration) extends DebugEnhancedLoggi
     trace(user)
     for {
       depositDir <- DepositDir.create(draftBase, user)
-      depositInfo <- depositDir.getDepositInfo(submitBase, easyHome)
+      depositInfo <- depositDir.getDepositInfo(depositPropertiesRepo, easyHome)
```

Comment:
I though we would only use graphql for the submitted bags, the drafts would stick to files. These properties belong to the draft.

Reply:
You are right, but in retrieving the `DepositInfo`, you need the `StateManager`, which might be looking into the submitted deposit properties (if the draft state is `SUBMITTED` or `IN_REVIEW`), which requires the GraphQL service.